docs/hopsworksai/aws/eks_ecr_integration.md

```diff
@@ -225,30 +208,31 @@ selfLink: /api/v1/namespaces/kube-system/configmaps/aws-auth
 uid: c794b2d8-9f10-443d-9072-c65d0f2eb552
 ```
 
-Once you are done with editing the configmap, save the updated config map.
+Once you are done with editing the configmap, save it and leave the editor. The output should be:
 
 ```bash
 configmap/aws-auth edited
 ```
 
 ##Step 4: Open Hopsworks required ports on your EKS cluster security group
 
-You need to open the HTTP (80) and HTTPS (443) ports on the security group of your EKS cluster.
-First, you need to get the name of the security group of your EKS cluster by using the following eksctl command. Notice that you need to change the cluster name according to your setup in Step 1 or if you have an existing cluster.
+We will run hopsworks in the virtual network of the EKS cluster. For hopsworks to run properly we recommend opening the ports for HTTP (80) and HTTPS (443). In this section, we will see how to do it. It is possible to not open these ports at the cost of some features. See [Limiting permissions](restrictive_permissions.md#step-1-create-a-vpc) for more details.
```

Comment:
For services/jobs running on the EKS cluster to be able to communicate with Hopsworks, you need to open the port for HTTPS (443).
It is possible to not open the port(s), but this will prevent jobs that run on the EKS cluster from talking directly with the Hopsworks REST API.
 
I don't think you need port 80, do you? I thought it was only for letsencrypt.

Reply:
Ok I guess the text is not clear. 
To avoid having to explain vpc peering here we run hopsworks directly in the VPC created by EKS, the result is that we need to open 443 and 80 for Hopsworks. It has nothing to do with the communication between EKS and Hopsworks it is all about letsencrypt and the hopsworks rest end point.